scripts/dev/check_parameters.php

```diff
@@ -372,6 +373,9 @@ function recurse($path)
     }
 }
 
+$GLOBALS['error_reported'] = false;
 foreach ($dirs as $dir) {
     recurse(realpath($dir));
 }
+
+exit($GLOBALS['error_reported'] === false ? 0 : 1);
```

Comment:
Maybe use `2` on failure, to distinguish that from the already existing `1` status codes?